### PR TITLE
Email tempaltes

### LINKED ISSUE #851: Message Templates

### PR DESCRIPTION
**Pull request by haiodo**
Message templates

Closes #851 

Screenshot 2022-01-22 at 16 58 30
Screenshot 2022-01-22 at 16 58 56

### CODE REVIEW

File: plugins/templates-resources/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//
+
+import { Resources } from '@anticrm/platform'
+import Templates from './components/Templates.svelte'
+import { TextEditorHandler } from '@anticrm/text-editor'
+import { showPopup } from '@anticrm/ui'
+import TemplatePopup from './components/TemplatePopup.svelte'
+
+function ShowTemplates (element: HTMLElement, editor: TextEditorHandler): void {
+  console.log('FSK')
```
Comment (BykhovDenis) [inline plugins/templates-resources/src/index.ts]: Extra log

File: plugins/workbench-resources/src/components/AccountPopup.svelte [hunk 2]
```diff
@@ -45,29 +43,31 @@
...
     const loc = getCurrentLocation()
     loc.path[1] = setting.ids.SettingApp
-    loc.path[2] = sp.id
+    loc.path[2] = sp.name
     loc.path.length = 3
     navigate(loc)
   }
 
   function signOut (): void {
+    closePopup()
```
Comment (BykhovDenis) [inline plugins/workbench-resources/src/components/AccountPopup.svelte]: Why? User will be redirected and popup will be destroyed. Or not?

**Review by BykhovDenis — APPROVED** [review]

File: plugins/workbench-resources/src/components/AccountPopup.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/workbench-resources/src/components/AccountPopup.svelte:56]: this change is not Belong to this PR any more :-)

File: plugins/templates-resources/src/index.ts (hunk 1 above)
Comment (haiodo) [reply plugins/templates-resources/src/index.ts]: Fixed.

File: plugins/workbench-resources/src/components/AccountPopup.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/workbench-resources/src/components/AccountPopup.svelte]: Fixed.

**Pull request merged**